code/style.css

```diff
@@ -1,4 +1,157 @@
+body{
+  margin:0;
+}
 html {
-  background: #0025ff;
-  color: #fff;
+  color: black;
+  font-family: Helvetica, sans-serif, Arial,;
+  font-size: 12px;
+  background-color: white;
+  
+}
+
+.logo {
+  width: 100px;
+  height: 100px;
+  position: sticky;
+  border-radius: 20px ;
+  margin: 30px;
+  box-shadow: 10px 5px 5px teal;
+
+}
+
+header{
+  width: 100%;
+  height: 200px;
+  background-color: white;
+
+}
+nav{
+  display:flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 20px;
+}
+.site-name{
+  font-size: 6vw;
+  position: center;
+
+}
+li{
+  padding: 10px;
+  position: -webkit-sticky;
+  border-right: 20px;
+}
+h2:hover {
+  background-color: #254e58;
+}
+
+.big-news{
+  display: flex;
+  flex-direction: row;
+  margin: 10px 20px;
+  padding: 20px;
+  margin-right:10px;
+  text-align: center;
+
+  background-color: rgb(250, 250, 250);
+  align-items: left;
+}
+
+.big-news-image{
+  width: 100%;
+  
+}
+.text-container{
+  display: flex;
+  flex-direction:column;
+
+}
+
+.big-news-text{
+  text-align: center;
+  margin: 5px;
+  flex-grow: 1;
+  justify-content: center;
+  padding: 10px;
+}
+
+.flex-box {
+  display: flex;
+  height: 50%;
+  width: 50%;
+
+}
+
+.flex-parent {
+
+  height: 800px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  flex-wrap: wrap;
+  border-width: 15 px;   
+} 
+   
+.flex-child {
+  margin: 15px;
+  border: 2px solid #254e58;
+  border-radius: 5px;
+  height: 300px;
+  width: 300px;
+  flex-grow: 1;
+  padding: 10px;
+
+  background-color: white;
+  animation-name: example;
+  animation-duration: 4s;
+  
+}
+@keyframes example {
+  0%   {background-color: white;}
+  25%  {background-color: rgb(162, 187, 113);}
+  50%  {background-color: rgb(76, 117, 76);}
+  100% {background-color: teal;}
+}
+
+.flex-child img {
+  background-size: contain;
+  justify-content: center;
+  align-items: center;
+  width: 300px;
+  height: 200px;
+}
+
+.p{
+  color: black;
+  text-align: right;
+}
+/*
+@keyframes example {
+  from {background-color: lightgray;}
+  to {background-color: white;}
+}*/
+
+
+@media (max-width:767px) {
+  #navigation-bar{
+    display:none;
+  } 
+  .header{
+    flex-direction: row;
+    text-align: right;
+  }
+  .big-news{
+    flex-direction: column;
+  }
+  .flex-parent{
+    grid-template-columns: repeat(2, 1fr);
+    gap: 0 20px;
+  }
 }
+@media (min-width:992px) {
```

Comment:
There is something on ipad view that makes the content align left instead, the margin on the right is much bigger. Can not fint the solution right now though.  Will try to fint it!

Reply:
Ah, it is the text that pushes all aside. So, a solution could be to change the flex-direction to column instead of row on the big-news. So the text will be under the img.  Otherwise you need to set some css to the img in smaller screens for the image to rezise down to give the text some space.

Reply:
My solution  would be to write in the mediaquery for the img in ipad view like this:
```
img.big-news-image {
    width: 50%;
}
```